docs/configuration.md

```diff
@@ -1506,6 +1506,11 @@ Apart from these, the following properties are also available, and may be useful
     of this setting is to act as a safety-net to prevent runaway uncancellable tasks from rendering
     an executor unusable.
   </td>
+  <td><code>spark.stage.maxConsecutiveAttempts</code></td>
+  <td>4</td>
+  <td>
+    Number of consecutive stage retries allowed before a stage is aborted (since you can have multiple fetch failures in one stage attempt)
```

Comment:
Oops sorry just noticing that I don't think @squito intended for the part in parens to be in the comment?  You could make this instead 

Number of consecutive stage retries allowed before a stage is aborted (stages are retried if shuffle fetch failures occur).

or, you could just eliminate the part in parens.  Sorry I didn't notice this earlier!